styles.css

```diff
@@ -2,21 +2,67 @@
     box-sizing: border-box;
 }
 
-body, html {
+body{
     margin: 0;
     padding: 0;
+    background-color: #424b52;
+    color: whitesmoke;
+    font-family: 'Lato', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
 }
 
-nav {
+/* nav {
```

Comment:
Not a big fan of leaving commented code in PRs. We have version control to recover old code.